google/cloud/aiplatform/models.py

```diff
@@ -980,6 +1011,12 @@ def _deploy_call(
                 "Both `accelerator_type` and `accelerator_count` should be specified or None."
             )
 
+        if not accelerator_type or not accelerator_count and autoscaling_target_accelerator_duty_cycle:
```

Comment:
I don't think this check is working as expected (I tried deploying a model with only `autoscaling_target_cpu_utilization` set and it raised this error). You can change it to something like:

```
if autoscaling_target_accelerator_duty_cycle is not None and (not accelerator_type or not accelerator_count):
```

Reply:
Fixed in https://github.com/googleapis/python-aiplatform/pull/1197/commits/9501eb0904ee10a5c7a35425cb21c732659c5a3c